Framework/Core/src/CommonServices.cxx

```diff
@@ -84,7 +84,9 @@ o2::framework::ServiceSpec CommonServices::monitoringSpec()
       o2::monitoring::Monitoring* monitoring;
       if (useDPL) {
         monitoring = new Monitoring();
-        monitoring->addBackend(std::make_unique<DPLMonitoringBackend>(registry));
+        auto dplBackend = std::make_unique<DPLMonitoringBackend>(registry);
+        (dynamic_cast<o2::monitoring::Backend*>(dplBackend.get()))->setVerbosisty(o2::monitoring::Verbosity::Debug);
+        monitoring->addBackend(std::move(dplBackend));
```

Comment:
@awegrzyn there is actually a typo (setVerbosisty) in the API.

Reply:
Yes, fixing, from v3.8.9 you will be able to use both.